README.md

```diff
@@ -26,6 +26,7 @@ A collection of awesome email newsletter tools, platforms, media, and software.
 
 ## Communities
 
+- [Newsletter Upstarts](https://bit.ly/newslttring) - Small group feedback meetups of newsletter owners, invite only. Fill in the form for invite.
```

Comment:
Hi there, is this link supposed to redirect to a Google form?

Reply:
Ok, I see it's an application form. Can you please put this in alphabetical order after Substack Writers?